docs/main/user-guides/local-storage-user-guide/local-pv-hostpath/advanced-operations/xfs-quota/modify-xfs-quota.md

```diff
@@ -0,0 +1,187 @@
+```
+NAME             STATUS   ROLES    AGE   VERSION
+storage-node-2   Ready    worker   10m   v1.22.1
+```
+
+## Modify the XFS Quota Limits
+
+You can change the soft and/or hard limit of an existing hostpath volume with XFS project quota enabled by following the steps below. If you wish to remove the XFS project quota entirely, refer to the [Remove Project](#remove-project) section.
+
+### Change Quota Limits
+
+Execute the following commands on the node where the hostpath volume exists:
+
+- Make a note of the Project ID.
+
+```
+$ sudo xfs_quota -x -c 'report -h' /var/openebs/local
+```
+
+**Example Output**
+
+```
+Project quota on /var/openebs/local (/dev/nvme1n1)
+                        Blocks
+Project ID   Used   Soft   Hard Warn/Grace
+---------- ---------------------------------
+#0           0      0      0   00 [------]
+#1           1G     2.0G   2.0G 00 [------]
+```
+
+- Modify the quota limits using the following command. The values for `bsoft` (soft limit) and `bhard` (hard limit) must be in B/KB/MB/GB (not KiB/MiB/GiB).
+
+```
+$ sudo xfs_quota -x -c 'limit -p bsoft=3G bhard=5G 1' /var/openebs/local
+```
+
+- Verify the updated limits.
+
+```
+$ sudo xfs_quota -x -c 'report -h' /var/openebs/local
+```
+
+**Example Output**
+
+```
+Project quota on /var/openebs/local (/dev/nvme1n1)
+                        Blocks              
+Project ID   Used   Soft   Hard Warn/Grace   
+---------- --------------------------------- 
+#0              0      0      0  00 [------]
+#1             1G     3G     5G  00 [------]
+```
+
+## Remove Project
+
+To completely remove the XFS project quota from a volume, follow these steps:
+
+1. Make a note of the Project ID.
+
+```
+$ sudo xfs_quota -x -c 'report -h' /var/openebs/local
+```
+
+**Example Output**
+
+```
+Project quota on /var/openebs/local (/dev/nvme1n1)
+                        Blocks              
+Project ID   Used   Soft   Hard Warn/Grace   
+---------- --------------------------------- 
+#0              0      0      0  00 [------]
+#1             1G   2.0G   2.0G  00 [------]
+```
+
+2. Set the project limits to 0, effectively removing the quota limits.
+
+```
+$ sudo xfs_quota -x -c 'limit -p bsoft=0 bhard=0 1' /var/openebs/local
+```
+
+:::note
+The command is for a project ID=1 at directory path `/var/openebs/local`.
+:::
+
+3. Clear the directory tree from the XFS project quota.
+
+```
+$ sudo xfs_quota 'project -C -p /var/openebs/local 1' /var/openebs/local
+```
+
+4. Verify the changes.
+
+```
+$ sudo xfs_quota -x -c 'report -h' /var/openebs/local
+```
+
+**Example Output**
+
+```
+Project quota on /var/openebs/local (/dev/nvme1n1)
+                        Blocks
+Project ID   Used   Soft   Hard Warn/Grace
+---------- ---------------------------------
+#0           1G     0      0   00 [------]
+```
+
+## See Also
+
+- [XFS Quota Prerequisites](xfs-quota-pre.md)
+- [Enable XFS Quota on LocalPV Hostpath](enable-xfs-quota.md)
+- [XFS Quota with Loop Device](xfs-quota-pre.md)
```

Comment:
These links may not work..

Reply:
I checked the links & all the links are working properly. @niladrih